rootfs/etc/confd/templates/builder

```diff
@@ -185,10 +185,9 @@ if [ $CODE -ne 0 ]; then
 fi
 
 RELEASE=$(echo $PUBLISH_RELEASE | extract-version)
-DOMAIN=$(echo $PUBLISH_RELEASE | extract-domain)
 indent "done, $APP_NAME:v$RELEASE deployed to Deis"
 echo
-indent "http://$DOMAIN"
+indent "Use \`deis open\` to view this application in your browser."
```

Comment:
Typically `-a` isn't necessary as when they're doing a `git push deis master`, they're in the correct directory.